common/display/src/main/scala/uk/ac/wellcome/display/models/AggregationRequest.scala

```diff
@@ -6,14 +6,28 @@ case class InvalidAggregationStringKeyRequest(key: String)
     extends InvalidStringKeyException
 
 case class AggregationsRequest(values: List[AggregationRequest])
+
 sealed trait AggregationRequest
-final case class WorkTypeAggregationRequest() extends AggregationRequest
 
 object AggregationRequest {
+
+  case object WorkType extends AggregationRequest
+
+  case class Date(interval: DateInterval = DateInterval.Year)
+      extends AggregationRequest
+
   def apply(str: String)
     : Either[InvalidAggregationStringKeyRequest, AggregationRequest] =
     str match {
-      case "workType" => Right(WorkTypeAggregationRequest())
-      case _          => Left(InvalidAggregationStringKeyRequest(str))
+      case "workType" => Right(AggregationRequest.WorkType)
+      case "year" =>
```

Comment:
Are we sure we want to go with year here and not `production.dates.period` and use the year as a default aggregation bucket size? 

Ping @jtweed